docs/zz_generated.seed.yaml

```diff
@@ -274,6 +274,8 @@ spec:
   # Optional: Detailed location of the cluster, like "Hamburg" or "Datacenter 7".
   # For informational purposes in the Kubermatic dashboard only.
   location: ""
+  # Metering enables the metering on user clusters across the seed.
+  metering: null
```

Comment:
Please extend https://github.com/kubermatic/kubermatic/blob/master/cmd/example-yaml-generator/main.go#L133 with an empty dummy object so the generated YAML will contain the various sub fields below `metering`.